rllib/offline/offline_data.py

```diff
@@ -223,7 +223,7 @@ def sample(
     @property
     def default_read_method_kwargs(self):
         return {
-            "override_num_blocks": max(self.config.num_learners * 2, 2),
+            # "override_num_blocks": max(self.config.num_learners * 2, 2),
```

Comment:
not sure I understand this change ...

Reply:
It shouldn't be there. Was done during testing. In the streaming optimization PR this will be removed from the default arguments and all examples.

Reply:
Great catch btw! :)